src/routes/safe/container/index.tsx

```diff
@@ -1,76 +1,87 @@
-import * as React from 'react'
-import { useState } from 'react'
+import { GenericModal } from '@gnosis.pm/safe-react-components'
+//import { makeStyles } from '@material-ui/core/styles'
```

Comment:
Do we still need the styles commented lines?

Reply:
nope